api/openapi/my_api.yaml

```diff
@@ -1,66 +1,154 @@
 openapi: 3.0.0
 # Added by API Auto Mocking Plugin
+servers: []
```

Comment:
I see the test failed due to connection issues. Wondering whether adding this servers line would have something to do with it. Suggest to remove this servers line and see if that would make any difference to the test failure.